framework/raven_qsub_command.sh

```diff
@@ -8,6 +8,7 @@ fi
 # conda definitions should be set up in raven/.ravenrc after installing with raven/scripts/establish_conda_env.sh
 module load raven-devel
 module load MVAPICH2/2.0.1-GCC-4.9.2
+module load imkl/10.3.6.233
```

Comment:
is this modification related to this PR?

Reply:
yes (it makes the cluster usage more robust)